lib/pages/chat/send_file_dialog/files_listview_widget.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:fluffychat/pages/chat/send_file_dialog/hover_actions_widget.dart';
+import 'package:fluffychat/presentation/list_notifier.dart';
+import 'package:fluffychat/resource/image_paths.dart';
+import 'package:fluffychat/utils/matrix_sdk_extensions/matrix_file_extension.dart';
+import 'package:fluffychat/widgets/file_widget/file_error_tile_widget_style.dart';
+import 'package:fluffychat/widgets/file_widget/file_tile_widget.dart';
+import 'package:fluffychat/widgets/file_widget/file_tile_widget_style.dart';
+import 'package:flutter/material.dart';
+import 'send_file_dialog_style.dart';
+
+import 'package:matrix/matrix.dart';
```

Comment:
pls update it full path

Reply:
using this actually better, become when you move file to another folder, it's got updated